ztp/kube-compare-reference/required/cluster-logging/ClusterLogForwarder.yaml

```diff
@@ -1,37 +1,26 @@
-apiVersion: "logging.openshift.io/v1"
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
   name: instance
   namespace: openshift-logging
 spec:
   {{- if .spec.outputs }}
   outputs:
-    - type: {{ (index .spec.outputs 0).type }}
-      name: {{ (index .spec.outputs 0).name }}
-      kafka:
-        url: {{ (index .spec.outputs 0).kafka.url }}
+  - name: {{ (index .spec.outputs 0).name }}
+    type: kafka
```

Comment:
@imiller0 as far as the kube-compare reference goes should we only allow kafka forwarder or any output?

Reply:
For now kafka as that is what we have evaluated the resource utilzation with. Anything outside of that would need additional verification in the RAN DU use case.